doc/install.rst

```diff
@@ -64,46 +64,18 @@ Installing the latest release
 
 Then run:
 
-.. raw:: html
+.. prompt:: bash $
 
-       <div class="highlight"><pre><code
```

Comment:
This changes the behavior of the buttons in the [install page](https://scikit-learn.org/dev/install.html#installing-the-latest-release).

Can we revert this change?

Reply:
Yes, sorry for the delay.

Reply:
Now that my computer issue is resolved, I have made the requested changes but I see that it has unresolved issues below. Should this change have been made on a new branch?

Reply:
This is resolved in my last change.